ByteCoin/Controller/ViewController.swift

```diff
@@ -19,12 +19,20 @@ class ViewController: UIViewController, CoinManagerDelegate, UIPickerViewDelegat
 	var coinManager = CoinManager()
 
 	override func viewDidLoad() {
-        super.viewDidLoad()
-        // Do any additional setup after loading the view.
+		super.viewDidLoad()
+		// Do any additional setup after loading the view.
```

Comment:
It looks like these are just whitespace changes; does Xcode have any auto-formatting features?